.tx/config

```diff
@@ -2,8 +2,7 @@
 host = https://www.transifex.com
 
 [o:open-edx:p:xblocks:r:lti-consumer]
-file_filter = lti_consumer/translations/<lang>/LC_MESSAGES/text.po
-source_file = lti_consumer/translations/en/LC_MESSAGES/text.po
+file_filter = lti_consumer/conf/locale/<lang>/LC_MESSAGES/text.po
```

Comment:
This will still extract translations to this repository. In the grand scheme of the project, is the idea that eventually the source files in this directory will be removed from this repository and will instead be committed to `openedx-translations`?

Reply:
Yes, it will extract it to this repo but the destination directory is ignored by `.gitignore`. daily Action is triggered in `openedx-translations` to collect translations from all supported repos, so the generated `text.po` will be copied to `openedx-translations` within during the Action execution